damus/Views/Relays/RelayConfigView.swift

```diff
@@ -130,16 +172,18 @@ struct RelayConfigView: View {
                 }
             }
             
-            if recommended.count > 0 {
-                Section {
-                    List(recommended, id: \.url) { r in
-                        RecommendedRelayView(damus: state, relay: r.url.id, showActionButtons: $showActionButtons)
+            Section {
+                if recommended.count > 0 {
```

Comment:
It's better practice to use this instead, because the full count of the array isn't needed:
```swift
if !recommended.isEmpty { ... }
```